layouts/partials/site-favicon.html

```diff
@@ -1,3 +1,3 @@
 {{ if .Site.Params.favicon }}
-<link rel="shortcut icon" href="/{{ .Site.Params.favicon }}" type="image/x-icon" />
-{{ end }}
\ No newline at end of file
+<link rel="shortcut icon" href="{{ relURL ($.Site.Params.favicon) }}" type="image/x-icon" />
```

Comment:
Thanks again. And again could you do as for #252 and remove the () :)

Thanks a ton!

Reply:
May I make it easier to change? :)
```suggestion
<link rel="shortcut icon" href="{{ relURL $.Site.Params.favicon }}" type="image/x-icon" />
```